editor.html

```diff
@@ -240,6 +240,17 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
             <div id="editor" class="flex3" tabindex="-1"></div>
           </div>
         </div>
+        <script type="text/javascript">
+        function script(url) {
+            var s = document.createElement('script');
+            s.type = 'text/javascript';
+            s.async = false;
+            s.defer = true;
+            s.src = url;
+            var x = document.getElementsByTagName('head')[0];
+            x.appendChild(s);
+        }
+        </script>
```

Comment:
Since the lazy loading is done in the python-main.js file can we move this function there?
It should probably be a global function though, no need to have it inside the pythonEditor or web_editor objects.

Reply:
Yep that makes more sense
WebUSB also uses the same function to load DAPjs and hterm when WebUSB is available, should I make the change in both or will that make a confusing git conflict?

Reply:
This will end up in master soonish, so webusb can be rebased and take it from here.

Reply:
If we are going to update the primary editor soon I would like to include this change as well.
Could you move this function to the bottom of the python-main.js file?

Afterwards we could merge master into the webusb branch (no rebases, as that would change the commit hashes and be a problem next time we merge webusb to beta-integration).